features/Extended Materials/Shaders/ExtendedMaterials/ExtendedMaterials.hlsli

```diff
@@ -311,45 +311,62 @@ namespace ExtendedMaterials
 		return coords;
 	}
 
-	// https://advances.realtimerendering.com/s2006/Tatarchuk-POM.pdf
+#if defined(TRUE_PBR)
+#	define SHADOW_INTENSITY 1.0
+#else
+#	define SHADOW_INTENSITY 2.0
+#endif
 
+	// https://advances.realtimerendering.com/s2006/Tatarchuk-POM.pdf
 	// Cheap method of creating shadows using height for a given light source
 	float GetParallaxSoftShadowMultiplier(float2 coords, float mipLevel, float3 L, float sh0, Texture2D<float4> tex, SamplerState texSampler, uint channel, float quality, float noise, DisplacementParams params)
 	{
-		[branch] if (quality > 0.0)
+		[branch] if (quality > 0.0 || extendedMaterialSettings.ExtendShadows)
 		{
-			float2 rayDir = L.xy * 0.1;
-			float4 multipliers = rcp((float4(4, 3, 2, 1) + noise));
+			float2 rayDir = L.xy * 0.1 * params.HeightScale;
+			float4 multipliers = rcp((float4(1, 2, 3, 4) + noise));
 			float4 sh;
-			sh.x = AdjustDisplacementNormalized(tex.SampleLevel(texSampler, coords + rayDir * multipliers.x, mipLevel)[channel], params);
-			sh.y = AdjustDisplacementNormalized(tex.SampleLevel(texSampler, coords + rayDir * multipliers.y, mipLevel)[channel], params);
-			sh.z = AdjustDisplacementNormalized(tex.SampleLevel(texSampler, coords + rayDir * multipliers.z, mipLevel)[channel], params);
-			sh.w = AdjustDisplacementNormalized(tex.SampleLevel(texSampler, coords + rayDir * multipliers.w, mipLevel)[channel], params);
-			return 1.0 - saturate(dot(max(0, sh - sh0), 1.0) * params.HeightScale * 2.0) * quality;
```

Comment:
Removed HeightScale from here as it makes more sense to me (and looks better) for it to scale the actual ray than to multiply the resulting angle against the light dir.